### PR TITLE
Remove motoring Fixed Penalty Notice

### PR DESCRIPTION
**Pull request by Davidslv**
A decision has been made to remove FPN from the service as these notices are spent on the day that they are given. 
This commit removes all the references to either Youth or Adult FPN convictions.

All done in one commit to be easier to revert (if ever needed).

Story: https://trello.com/c/Vcpqhlqr

### CODE REVIEW

**Comment by Davidslv** [comment]
[[DC] Legal Q2 - Understand and agree work for removal of FPNs](https://trello.com/c/Vcpqhlqr/48-dc-legal-q2-understand-and-agree-work-for-removal-of-fpns)

File: config/locales/en/helpers.yml [hunk 1]
```diff
@@ -241,13 +239,9 @@ en:
           adult_sexual_harm_prevention_order: A court ruled that you pose a risk of causing a sexual offence or sexual harm
           # adult_motoring
           adult_disqualification: You were banned from driving
-          adult_motoring_fine: A court gave you a fine that was not a fixed penalty notice (FPN)
```
Comment (zheileman) [inline config/locales/en/helpers.yml:244]: Beware, I think `adult_motoring_fine` was removed by mistake. It belongs to motoring fines that we still want in the service.

File: config/locales/en/helpers.yml [hunk 2]
```diff
@@ -241,13 +239,9 @@ en:
...
-          adult_penalty_notice: You were given a fine ‘on the spot’ or by post
           adult_penalty_points: You were given a number of penalty points for a driving offence
           # youth motoring
           youth_disqualification: You were banned from driving
-          youth_motoring_fine: A court gave you a fine that was not a fixed penalty notice (FPN)
```
Comment (zheileman) [inline config/locales/en/helpers.yml:249]: Same here.

**Review by zheileman — COMMENTED** [review]
👍 
It needs a rebase because what I merged yesterday touches some files/lines you've touched here so there will be some conflicts.

**Review by zheileman — APPROVED** [review]

**Pull request merged**